src/main/java/com/lukian/bookstore/service/user/UserServiceImpl.java

```diff
@@ -44,6 +46,9 @@ private void assignDefaultUserRoleToNewUser(User user) {
                         .orElseThrow(() -> new EntityNotFoundException(
                                 "Cannot find role by given name"));
         user.getRoles().add(roleFromDb);
+        Set<Role> roles = new HashSet<>();
+        roles.add(roleFromDb);
+        user.setRoles(roles);
     }
 
```

Comment:
What this changes doing here in PR of Docker?

Reply:
I was testing app in container and stumbled upon issue during user registration (npe). But yeah, it is not related to docker, I should have done it in separate fix branch